docs/pages/blog/nextgen-pickers.md

```diff
@@ -0,0 +1,130 @@
+---
+title: A fresh approach for date and time inputs
+description: Check out what's new with the next-gen pickers in earlier v6 versions.
+date: 2022-12-20T00:00:00.000Z
+authors: ['josefreitas']
+tags: ['MUI X', 'News']
+---
+
+There's a lot of exciting news in MUI X v6, but there's hardly anything comparable to the revamp we're delivering for the [Date and Time Pickers](https://next.mui.com/x/react-date-pickers/getting-started/).
+
+It's the result of our research and effort to improve these inputs' usability and overall customizability.
+We kept the best aspects of the previous versions, and we're fleshing out the package with several improvements and one fundamental change: a new input type to replace the masked text fields.
+
+## The new date and time fields
```

Comment:
As a general rule, I think that the blog posts that get people excited always focus the majority of the attention on the problem, they never put the solution at the center.

In this post, I think that we could focus more on seeing a clear demonstration of the problems. For example, this is a horrible UX, we could cover it:

https://user-images.githubusercontent.com/3165635/210186164-ef38aecf-d0e4-412f-914f-f71b9e7e3ec8.mov

https://mui.com/x/react-date-pickers/date-picker/#basic-usage WUT is going on with the year? WUT is the field in the error stage, I'm still editing?